src/operator/nn/mkldnn/mkldnn_act-inl.h

```diff
@@ -67,8 +63,28 @@ MKLDNNActForward &GetActForward(const ActivationParam& param,
 void MKLDNNActivationForward(const nnvm::NodeAttrs& attrs, const OpContext &ctx,
                              const NDArray &in_data, const OpReqType &req,
                              const NDArray &out_data);
+
+mkldnn::eltwise_backward::primitive_desc GetActBwdDescImpl(
+    const ActivationParam &param, const mkldnn::memory &input_mem,
+    const mkldnn::memory &diff_dst_memory);
+
+class MKLDNNActBackward {
+  std::shared_ptr<mkldnn::eltwise_backward> bwd;
```

Comment:
Make this as a private member? As there's an API `GetBwd()` to retrieve it.

Reply:
Thanks. Will change it.